environment.yml

```diff
@@ -1,8 +1,25 @@
-name: mag-0.1.0
+name: nf-core-mag-1.0.0
 channels:
   - bioconda
   - conda-forge
   - defaults
 dependencies:
-  - fastqc=0.11.7
-  - multiqc=1.5
+  - fastqc=0.11.8
```

Comment:
I'm kind of surprised that you don't need more here... Most other pipelines have failed tests with just this line. See [methylseq](https://github.com/nf-core/methylseq/blob/1f0eaa6f1a8342e98c87b639553f54c47536f9c3/environment.yml#L9-L13) for an example of the full list of prerequisites as it stands.

Reply:
Though if it works, no need to change. But check that it does definitely work :)